src/axom/sidre/examples/sidre_stressgroups.cpp

```diff
@@ -80,7 +80,7 @@ int main(int argc, char* argv[])
   std::random_shuffle(names.begin(), names.end());
 
   axom::utilities::Timer create_timer(true);
-  for(unsigned int i = 0; i < num_groups; ++i)
+  for(int i = 0; i < num_groups; ++i)
```

Comment:
since names type is std::vector<std::string>, isn't size_t a better choice for type of i?

Reply:
This was a change to fix a warning error about comparing signed and unsigned values, `i < num_groups`.  `num_groups` is `IndexType` and assigned from a command line argument. I could change it to `size_t` all around. But `int` makes more sense to me since there will never be that many groups in the test.

Reply:
I changed `num_groups` to `size_t`.